skins/oasis/modules/LatestActivityController.class.php

```diff
@@ -56,29 +54,31 @@ public function executeIndex() {
 				$item['changetype'] = $change['type'];
 				$title = Title::newFromText( $change['title'], $change['ns'] );
 
-				if( !empty($change['articleComment']) && !empty($change['url']) ) {
-					$item['page_href'] = Xml::element('a', array('href' => $change['url']), $item['page_title']);
+				if ( !empty( $change['articleComment'] ) && !empty( $change['url'] ) ) {
+					$item['page_href'] = Xml::element( 'a', [ 'href' => $change['url'] ], $item['page_title'] );
 				} else {
-					if( is_object($title) ) {
+					if ( is_object( $title ) ) {
 						$item['page_url'] = $title->getLocalURL();
-						$item['page_href'] = Xml::element('a', array('href' => $item['page_url']), $item['page_title']);
+						$item['page_href'] = Xml::element( 'a', [ 'href' => $item['page_url'] ], $item['page_title'] );
 					}
 				}
-				
-				switch ($change['type']) {
+
+				switch ( $change['type'] ) {
 					case 'new':
 					case 'edit':
 					case 'delete':
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this "switch/case" number of lines from 15 to at most 10, for example by extracting code into function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1151)